packages/core/src/operation.ts

```diff
@@ -3,6 +3,8 @@ import type { Task } from './task';
 import type { Labels } from './labels';
 import type { FutureLike } from './future';
 
+export const toOperation = Symbol.for('toOperation');
```

Comment:
I think that we should use the converntions for symbol names even I the name of the variable isn't `Symbol.operation`

```suggestion
export const toOperation = Symbol.for('Symbol.operation');
```